src/pages/NewChatPage.js

```diff
@@ -169,7 +170,7 @@ function NewChatPage(props) {
             props.personalDetails,
             props.betas,
             searchTerm,
-            [],
+            selectedOptions,
```

Comment:
This caused a regression https://github.com/Expensify/App/issues/26072. We used to pass an empty array so we get a non-empty `userToInvite`. The `userToInvite` was used in `getHeaderMessage` not to return unwanted error messages.